zerver/views/webhooks/updown.py

```diff
@@ -0,0 +1,82 @@
+# Webhooks for external integrations.
+from __future__ import absolute_import
+import re
+from datetime import datetime
+from django.utils.translation import ugettext as _
+from zerver.lib.actions import check_send_message
+from zerver.lib.response import json_success, json_error
+from zerver.decorator import REQ, has_request_variables, api_key_only_webhook_view
+
+
+SUBJECT_TEMPLATE = "{service_url}"
+
+
+class UnsupprotedUpdownEventType(Exception):
```

Comment:
Also, I think maybe you should inherit from JsonableError so that this turns into a json_error message sent back to the user.